src/mcts/search.cc

```diff
@@ -682,7 +682,25 @@ void SearchWorker::ExtendNode(Node* node) {
       return;
     }
   }
-
+  
+  //WDL probing
+  if(board.our + board.their <= SyzygyTablebase::max_cardinality && board.GetNoCapturePly == 0
```

Comment:
Note that in porting tbprobe.h to lc0 I changed quite a few things.

You'll need an instance of SyzygyTablebase constructed and init'ed elsewhere available here.
instance->max_cardinality()

I also added a count method to bitboard so you can do (board.ours + board.theirs).count().

Reply:
that `count()` is what I'm calling `popcount()` in my other comment.